chef_master/source/custom_resources.rst

```diff
@@ -546,9 +546,9 @@ For example, a custom resource defines two properties (``content`` and ``path``)
    property :content, String
    property :path, String, name_property: true
```

Comment:
Would this work if path had the identify: true option and no block argument was used?

Reply:
That would show the other method I think works for this.